frontend/public/components/resource-dropdown.tsx

```diff
@@ -137,11 +143,17 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
       menuClassName="dropdown-menu--text-wrap"
       className={classNames('co-type-selector', className)}
       items={allItems}
-      title={allItems[selectedKey]}
+      title={
+        <div key="title-resource">
+          Resources{' '}
+          <Badge isRead>
+            {selected.length === 1 && selected[0] === 'All' ? 'All' : selected.length}
```

Comment:
There is a lot of `All`, I'm a bit worried if we are depending on that for display.

Reply:
I can switch that to an enum.